camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeperPlanner.java

```diff
@@ -86,4 +86,14 @@ protected String pathListToCommaSeperated(List<Path> list) {
   public Set<Properties> getOutlierProperties() {
     return this.outlierProperties;
   }
+
+  /**
+   * Blocks processing of a job until the input is ready.
+   * By default, will return immediately and proceed with the job.
+   * @param jobProps Job properties.
+   * @return true to proceed with the job, false to cancel the job.
+   */
+  public boolean waitUntilReadyToProcess(Properties jobProps) {
```

Comment:
Is this method going to be overridden in a subclass to provide a different behavior? This method can be protected if is not supposed to be overridden outside the package.